test/support/comment.ex

```diff
@@ -7,6 +7,6 @@ defmodule ExSieve.Comment do
 
     field :body
 
-    timestamps
+    timestamps()
```

Comment:
There are other places with this warning:

priv/repo/migrations/2_create_posts.exs
priv/repo/migrations/1_create_users.exs